e2e/scaling/input/namespace_default_system.nomad.hcl

```diff
@@ -4,6 +4,11 @@
 job "system_job" {
   type = "system"
 
+  constraint {
+    attribute = "${attr.kernel.name}"
+    value     = "linux"
+  }
```

Comment:
Note to reviewers: this is effectively already the case, but having it here explicitly communicates it to anyone who might run into this later.